parsingFunctions.js

```diff
@@ -0,0 +1,35 @@
+const parse = require('csv-parse/lib/sync');
+const XML = require('pixl-xml');
+const moment = require('moment');
+
+function csvParse(dataNum) {
+    return parse(dataNum, {columns:true});
+}
+function jsonParse(dataNum) {
+    let dataName = JSON.parse(dataNum);
+    let newDataName = [];
+    for (let i = 0; i < dataName.length; i++) {
+        newDataName.push({});

```

Comment:
Declaring an object literal would be nicer here:

    newDataName.push({
      Date: moment(...).format(...),
      From: dataName[i].FromAccount,
      ...